### ISSUE TITLE
Possible bug in Penryn 1f kernels

### ISSUE THREAD

**Issue by devinamatthews** [flame/blis#211]
GCC reports:

```
kernels/penryn/1f/bli_axpyf_penryn_int.c:98:33: warning: passing argument 1 of ‘bli_is_unaligned_to’ makes pointer from integer without a cast [-Wint-conversion]
            bli_is_unaligned_to( lda*sizeof(double), 16 ) )
                                 ^
In file included from kernels/penryn/1f/bli_axpyf_penryn_int.c:35:0:
./include/x86_64/blis.h:4004:15: note: expected ‘void *’ but argument is of type ‘long unsigned int’
 static bool_t bli_is_unaligned_to( void* p, siz_t size )
               ^
```

The same occurs for `dotxaxpyf` and `dotxf`.

**Comment by fgvanzee** [comment]
This likely became an issue recently, when I converted most of the macros to static functions.

I'll take a look at it and try to make the compiler happy.

**Comment by devinamatthews** [comment]
I guess it "probably works" on x86_64 but could cause problems elsewhere.

**Comment by fgvanzee** [comment]
Should be fixed now in 5140ee3.
